cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,103 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
 
-  });
+import registratePageObject from '../support/pages/registrate.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
 
-  beforeEach(() => {
+const registrate = new registratePageObject();
+const settings = new SettingsPageObject();
+const homePage = new homePageObject();
+const faker = require('faker');
 
-  });
+const userData = {
+  usernameData: faker.name.firstName().toLowerCase(),
+  userBio: 'Burning ass',
+  emailData: faker.internet.email(),
+  passwordData: faker.internet.password()
+};
+
+describe('Settings page', () => {
+  
+  beforeEach(() => {
+    cy.task('db:clear');
+  })
 
   it('should provide an ability to update username', () => {
+    cy.login();
```

Comment:
The 'login' method seems to be missing arguments. If it requires user data then it should be passed in as arguments.